### PR TITLE
LRDOCS-7402 CCR - SME #2

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LRDOCS-7402

Hey Russ,

I've only made minor edits. Looking forward to your final version after you finished editing the soft deletes doc.

Thanks,
Tibor

### CODE REVIEW

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/configuring-ccr-enabling-soft-deletes-on-elasticsearch-6.md [hunk 1]
```diff
@@ -6,17 +6,16 @@ For Elasticsearch 6 versions that support Cross-Cluster Replication (6.7+), you
 
 ## Step 1: Enabling Soft Deletes on the System and Company Indexes
 
-The system (`liferay-0`) and company (`liferay-[companyId]`) indexes can be soft delete-enabled by adding one line to the _Additional Index Configurations_ setting in Control Panel &rarr; Configuration &rarr; System Settings &rarr; Elasticsearch [Version]:
+You can enable soft deletes on the system (`liferay-0`) and company (`liferay-[companyId]`) indexes by adding one line to the _Additional Index Configurations_ setting in Control Panel &rarr; Configuration &rarr; System Settings &rarr; Elasticsearch [Version]:
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/liferay-enterprise-search/configuring-ccr-enabling-soft-deletes-on-elasticsearch-6.md:9]: _can be soft delete-enabled_ sounds weird so I tried to rephrase this.

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/cross-cluster-replication.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Read this article for an overview of the steps, then see the next article, [a st
 
 Any Liferay DXP node that will read from a local cluster's follower indexes and write through a separate connection to the remote cluster's leader indexes, must have the CCR module installed. This module is available with your LES subscription.
 
-Though all DXP nodes will have the CCR module installed, those that read and write from the same cluster only need to configure a normal REMOTE mode connection to the Elasticsearch cluster.
+Those DXP cluster nodes that read and write from the same cluster only need to configure a normal REMOTE mode connection to the Leader Elasticsearch cluster.
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/liferay-enterprise-search/cross-cluster-replication.md:31]: The first paragraph is saying the only the local-follower DXP cluster nodes must have the CCR module. But the 2nd section starts with saying "all DXP nodes will have", which is a contradiction.

The DXP leader nodes don't need the CCR module to be installed.

**Comment by lipusz** [comment]
Regarding this: https://github.com/rbohl/liferay-learn/pull/10#issuecomment-634272277
> The _Configuring Cross-Cluster Replication_ article has gotten out of hand (>2,000 words). I'm thinking of ways to break it up:
> 
> By Data Center
> 
>     * Configuring the Remote Elasticsearch Cluster and Co-Located Liferay DXP Node
> 
>     * Configuring the Local Elasticsearch Cluster and its Co-Located Liferay DXP Node
>       [Titles just for illustration]
> 
> 
> By Product:
> 
>     * Configuring Elasticsearch for Cross-Cluster Replication
> 
>     * Configuring Liferay DXP for Cross-Cluster Replication
> 
> 
> I'm not going to do this before sending it back to you, just curious what you think of either approach?

The first approach looks better to me, because there is also a certain order in which the steps can be made and that more-or-less limits how it can be split in a logical way.

1.1. First you configure the Leader ES cluster 
1.2. Then you configure the DXP node(s) that connect(s) to the Leader ES cluster. Start so indexes are created, checkpoint 1.
2.1 Configure the Follower ES cluster
2.2 Configure the DXP node(s) that connect(s) to the Follower ES cluster. Start and check if indexes are indeed replicated. Checkpoint 2.

**Comment by lipusz** [comment]
I was thinking about what else we could add to the Troubleshooting section. I checked the Elastic Discuss forum and maybe we could link https://discuss.elastic.co/tag/cross-cluster-replication and suggest customers to checkout those conversations. Maybe someone else has already asked the same question or run into the same issue.

WDYT? :thinking:

**Comment by lipusz** [comment]
Some of the troubleshooting sections in https://grow.liferay.com/excellence/LES+Cross+Cluster+Replication+CCR+Support+Guide#troubleshooting could also be added to the public docs.

File: docs/dxp/7.x/en/using-search/liferay-enterprise-search/configuring-ccr-enabling-soft-deletes-on-elasticsearch-6.md (hunk 1 above)
Comment (rbohl) [reply docs/dxp/7.x/en/using-search/liferay-enterprise-search/configuring-ccr-enabling-soft-deletes-on-elasticsearch-6.md:9]: Nice--let's go even more active: _Enable soft deletes on the system and company indexes by adding..._

**Comment by rbohl** [comment]
Maybe we make troubleshooting enhancements in phase 2?
https://issues.liferay.com/browse/LRDOCS-7764

**Comment by lipusz** [comment]
At least a minimal troubleshooting guide could be added now.

**Pull request merged**